technique/muon/tests/data/corrupted_plot_data_01.csv

```diff
@@ -0,0 +1,602 @@
+# run_number:00000039
```

Comment:
Usually we shouldn't have large data files for a unit test.

Can you remove this and split the unit test you've added into separate tests for the different cases:
- Date format wrong
- No data present
- Wrong number of fields
- etc